zebra-chain/Cargo.toml

```diff
@@ -25,7 +25,7 @@ hex = "0.4"
 jubjub = "0.6.0"
 lazy_static = "1.4.0"
 primitive-types = "0.9.0"
-rand_core = "0.5.1"
+rand_core = "0.6"
```

Comment:
Does this add a dependency on multiple versions of `rand_core`?

Maybe we should check which forks of ed25519-zebra use `rand_core = "0.6"` now. But this isn't a high priority.